HometaskPurchaseInStore/HWPurchaseInStore/App.cs

```diff
@@ -0,0 +1,64 @@
+﻿using HWPurchaseInStore.Models;
+using HWPurchaseInStore.Services;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace HWPurchaseInStore
+{
+    internal class App
+    {
+        public void RunApp()
+        {
+            GoodsInStore goodsInStore = new GoodsInStore();
```

Comment:
It's better to create class Good where will be property Name. Than you will need to add GoodService and add method GetGoogs(); from this method you will return list of Good with the corresponding names